manifests/service/cors/configure.pp

```diff
@@ -81,9 +78,6 @@
         owner   => $owner,
         group   => $group,
         content => template('lightblue/properties/lightblue-cors.json.erb'),
-        before  => $before,
-        notify  => $notify,
-        require => $require,
```

Comment:
Yes, over in #87 I updated the configure.pp and rspec tests. So now the class no longer defines the parameters, but the File definition is still able to use them, as proven by the new rspec test.

I am still looking for what the best practices are for this. I removed $before altogether, as the define should be able to handle that (and presumably $require). But I am still passing $notify to the File.